share/completions/mtr.fish

```diff
@@ -0,0 +1,41 @@
+complete -c mtr -x -a "(__fish_print_hostnames)"
+
+complete -c mtr -s h -l help -d 'display this help and exit'
+complete -c mtr -s v -l version -d 'output version information and exit'
+complete -c mtr -s 4 -d 'use IPv4 only'
+complete -c mtr -s 6 -d 'use IPv6 only'
+complete -c mtr -s F -l filename -d 'read hostname(s) from a file' -rF
+complete -c mtr -s r -l report -d 'output using report mode'
+complete -c mtr -s w -l report-wide -d 'output wide report'
+complete -c mtr -s x -l xml -d 'output xml'
+complete -c mtr -s t -l curses -d 'use curses terminal interface'
+complete -c mtr -l displaymode -d 'select initial display mode' -x -a '0 1 2'
+complete -c mtr -s g -l gtk -d 'use the GTK+ interface (if available)'
+complete -c mtr -s l -l raw -d 'output raw format'
+complete -c mtr -s C -l csv -d 'output comma separated values'
+complete -c mtr -s j -l json -d 'output json'
+complete -c mtr -s p -l split -d 'split output'
+complete -c mtr -s n -l no-dns -d 'do not resolve host names'
+complete -c mtr -s b -l show-ips -d 'show IP numbers and host names'
+complete -c mtr -s o -l order -d 'select output fields' -x
+complete -c mtr -s y -l ipinfo -d 'select IP information in output' -x -a '0 1 2 3 4'
+complete -c mtr -s z -l aslookup -d 'display AS number'
+complete -c mtr -s i -l interval -d 'ICMP echo request interval' -x
```

Comment:
This interval is given in *seconds*, I'd add that to the description.

Also maybe add some numbers? `-a '(seq 1 20)'`? Makes it clear that this expects numbers.

(same goes for the other options that take numbers here)

Reply:
@faho
But add `-a "(seq NUM NUM)"` may misleading users, let user think the available values are limited to this range.

Reply:
I will add description to no range limited arguments, and add `-a seq` to range limited arguments.